### PR TITLE
Pull Request #1 - Merging javascript changes to master (my own repo)

### PR DESCRIPTION
**Pull request by stepbystepdayo**
im so stupid! 
please check my code, bitch!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,33 +1,56 @@
 /************************************************************** Task 1: Warm-up! **************************************************************/
 //Task a: declare a variable called votingAge, console log true if age > 18 (no function required)
 
+let age = '20';
 
+if(age > 18){
+    console.log('true');
+}
```
Comment (tryingtokeepup) [inline index.js:10]: Good job here, checking for age variable!

File: index.js [hunk 2]
```diff
@@ -1,33 +1,56 @@
...
 
 //Task b: declare a variable and then use a conditional to change the value of that variable based on the value assigned to a second variable (no function required)
 
+let body = 'good';
+
+if(body === 'good'){
+    console.log('yoo can work out today!');
+}else if(body === 'bad' || body === 'not good'){
+    console.log('you have to rest today!');
+}
```
Comment (tryingtokeepup) [inline index.js:20]: its good to rest your body when you work out a lot!

Comment (tryingtokeepup) [reply index.js:20]: good job using `===` as your equality; two equals doesn't check for "type".

File: index.js [hunk 3]
```diff
@@ -1,33 +1,56 @@
...
 
 
 
 //Task c: Convert string ("1999") to integer (1999)  (no function required) // hint look up the Number method
 
+let era = '1999';
+let era2 = 1999;
+
+console.log(era2);
+
```
Comment (tryingtokeepup) [inline index.js:31]: this is not correct:

era2 = Number(era)

File: index.js [hunk 4]
```diff
@@ -60,19 +107,33 @@
...
 
+ var handGame = (Math.random());
+     if(handGame < 0.34){
+         console.log('Rock');
+     }else if(handGame <= 0.67){
+         console.log('Paper');
+     }else{
+         console.log('Sissers');
+     }
+
```
Comment (tryingtokeepup) [inline index.js:118]: not 'sissers', 'Scissors'

good job with the programming, though! :)

**Review by tryingtokeepup — COMMENTED** [review]
Great work, check your `Number()` function!

File: index.js (hunk 3 above)
Comment (stepbystepdayo) [reply index.js:31]: Number('1999');

**Pull request merged**